lib/brainstem/concerns/presenter_dsl.rb

```diff
@@ -142,13 +138,8 @@ def sort_order(name, *args, &block)
         #   @yieldreturn [ActiveRecord::Relation] A new scope that filters the
         #     scope that was yielded.
         #
-        def filter(name, type = DEFAULT_FILTER_DATA_TYPE, options = {}, &block)
-          if type.is_a?(Hash)
```

Comment:
It looks like we also didn't remove specs associated with this control flow. Are we thinking we're missing a bit of coverage?

Reply:
Specs have been removed for the deprecated behavior in `presenter_dsl_spec`